test/utils/assertions.js

```diff
@@ -0,0 +1,11 @@
+var _ = require('lodash');
+var path = require('path');
+var should = require('should');
+should.Assertion.add('equalFileSystemPath', function(value, description){
+    value = value.replace(/\//g, path.sep);
+    if(process.platform == 'win32' && _.startsWith(value, path.sep)){
+        value = __dirname.split(path.sep)[0] + value;
```

Comment:
I don't think that it is correct solution.
1. It is for absolute paths, but we compare only relative in tests
2. It takes disk name from current dir `__dirname` (`c:`, `d:`, etc), but it is not always correct (for example if we compare __dirname (with `c:`) with filename on another disk `f:`)